doc/input-output-reference/src/overview/group-design-objects.tex

```diff
@@ -884,7 +884,7 @@ \subsubsection{Inputs}\label{inputs-012}
 
 \item  \emph{Maximum} means that the maximum flow derived from \emph{Outdoor Air Flow per Person,} \emph{Outdoor Air Flow per Area, Outdoor Air Flow per Zone,} and \emph{Air Changes per Hour} (using the associated conversions to m\(^{3}\)/s for each field) will be used as the zone outdoor air flow rate.
    
-\item  \emph{IndoorAirQualityProcedure} means that the program will use the other procedure defined in ASHRAE Standard 62.1-2007 to calculate the amount of outdoor air necessary in order to maintain the levels of indoor air carbon dioxide at or below the setpoint defined in the ZoneControl:ContaminantController object. Appendix A of the ASHRAE 62.1-2010 user’s manual discusses another method for implementing CO2-based DCV in a single zone system. The last two methods of Proportional Control calculate the required outdoor air flow rate which varies in proportion to the percentage of the CO2 signal range and has two choices to calculate occupancy-based outdoor air rate. 
+\item  \emph{IndoorAirQualityProcedure} means that the program will use the other procedure defined in ASHRAE Standard 62.1-2007 to calculate the amount of outdoor air necessary in order to maintain the levels of indoor air carbon dioxide at or below the setpoint defined in the ZoneControl:ContaminantController object. Appendix A of the ASHRAE 62.1-2010 userâ€™s manual discusses another method for implementing CO2-based DCV in a single zone system. The last two methods of Proportional Control calculate the required outdoor air flow rate which varies in proportion to the percentage of the CO2 signal range and has two choices to calculate occupancy-based outdoor air rate. 
```

Comment:
There's still a funny character in here, but I'm assuming there's not enough need to push more commits solely for this.

![image](https://user-images.githubusercontent.com/849824/30810776-c82d4c26-a1cb-11e7-9792-1270be132f34.png)

Reply:
Whacky - you know when I made the doc changes last night, the local commit diff showed this line and another even though I hadn't changed anything, so I thought maybe my editor had automatically done something it shouldn't.  So, I undid those before I pushed it up. <sigh>